mne_bids/write.py

```diff
@@ -821,6 +882,22 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         inferred from the stim channel using `mne.find_events`.
     event_id : dict | None
         The event id dict used to create a 'trial_type' column in events.tsv
+    anonymize : dict | None
```

Comment:
I think you should add the caveat somewhere that all the times are assumed to be in UTC.

Reply:
I don't think that has to do with this function. This operates in whatever timezone meas_date is in.